extensions/indexes/range/src/org/exist/indexing/range/RangeIndexConfig.java

```diff
@@ -96,8 +97,18 @@ private void parseChildren(NodeList configNodes, Map<String, String> namespaces)
                     } else {
                         idxConf.add(newConfig);
                     }
-                } catch (DatabaseConfigurationException e) {
-                    LOG.error("Invalid range index configuration: " + e.getMessage());
+                } catch (final DatabaseConfigurationException e) {
+                    String uri = null;
+                    final Document doc = node.getOwnerDocument();
+                    if(doc != null) {
```

Comment:
I'd prefer a style like `String uri = (doc == null) ? null : doc.getDocumentURI();` but that is personal. dont change anything